components/eam/src/physics/cam/zm_conv.F90

```diff
@@ -2480,7 +1845,9 @@ subroutine cldprp(lchnk   , &
    end do
    do i = 1,il2g
       if (j0(i) < jb(i)) then
-         if (f(i,j0(i)) < 1.E-6_r8 .and. f(i,j0(i)+1) > f(i,j0(i))) j0(i) = j0(i) + 1
+         if (f(i,j0(i)) < 1.E-6_r8 .and. f(i,j0(i)+1) > f(i,j0(i))) then
```

Comment:
Just to note that this change goes the opposite way of the changes elsewhere that contain only a single statement within the if-endif block. Not really requesting to revert it.

Reply:
yea, I might have been a little inconsistent in this respect. In this particular case I was thinking the block is more readable when broken into 2 lines. I can see the argument that it's better to use single lines wherever possible though.